plugins/module_utils/vmware.py

```diff
@@ -50,8 +50,63 @@ class TaskError(Exception):
     def __init__(self, *args, **kwargs):
         super(TaskError, self).__init__(*args, **kwargs)
 
+def check_answer_question_status(vm):
+    """Check whether locked a virtual machine.
 
-def wait_for_task(task, max_backoff=64, timeout=3600):
+    Args:
+        vm: Virtual machine management object
+
+    Returns: bool
+    """
+    if hasattr(vm, "runtime") and vm.runtime.question:
+        return True
+
+    return False
+
+def make_answer_response(vm, answers):
+    """Make the response contents to answer against locked a virtual machine.
+
+    Args:
+        vm: Virtual machine management object
+        answers: Answer contents
+
+    Returns: Dict with answer id and number
+    Raises: TaskError on failure
+    """
+    response_list = {}
+    for message in vm.runtime.question.message:
+        response_list[message.id] = {}
+        for choice in vm.runtime.question.choice.choiceInfo:
+            response_list[message.id].update({
+                choice.label: choice.key
+            })
+
+    responses = []
+    try:
+        for answer in answers:
+            responses.append({
+                "id": vm.runtime.question.id,
+                "response_num": response_list[answer["question"]][answer["response"]]
+            })
+    except Exception:
+        raise TaskError("not found %s or %s or both in the response list" % (answer["question"], answer["response"]))
+
+    return responses
+
+def answer_question(vm, responses):
+    """Answer against the question for unlocking a virtual machine.
+
+    Args:
+        vm: Virtual machine management object
+        responses: Answer contents to unlock a virtual machine
+    """
+    for response in responses:
+        try:
+            vm.AnswerVM(response["id"], response["response_num"])
+        except Exception as e:
+            raise TaskError("answer failed: %s" % to_text(e))
+
+def wait_for_task(task, max_backoff=64, timeout=3600, vm=None, answers=None):
```

Comment:
```suggestion

def wait_for_task(task, max_backoff=64, timeout=3600, vm=None, answers=None):
```

linters: E302 expected 2 blank lines, found 1